### PR TITLE
[Shropshire] Shropshire styles and templates and initial Shropshire.pm file

### PR DESCRIPTION
**Pull request by MorayMySoc**
Adding styles and templates
Adding Shropshire PM file

Please check the following:

[skip changelog]

### CODE REVIEW

File: templates/web/shropshire/index-steps.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<h2>[% loc('How to report a problem') %]</h2>
+
+<ol class="big-numbers">
+    <li>[% question %]</li>
+    <li>Plot the location on the map</li>
+    <li>[% loc('Enter details of the problem') %]</li>
+    [% IF c.cobrand.is_council %]
```
Comment (dracos) [inline templates/web/shropshire/index-steps.html:7]: It is.

File: templates/web/shropshire/index-steps.html [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+        <li>Confirm the report and [% c.cobrand.council_name %] will investigate</li>
+    [% ELSIF c.cobrand.moniker == 'zurich' %]
```
Comment (dracos) [inline templates/web/shropshire/index-steps.html:9]: It is not.

**Review by dracos — CHANGES_REQUESTED** [review]
A template can be simplified, and the example places can be removed again :)

File: templates/web/shropshire/index-steps.html (hunk 1 above)
Comment (dracos) [reply templates/web/shropshire/index-steps.html:7]: (so this IF can be removed, I meant)

**Review by dracos — CHANGES_REQUESTED** [review]
One IF can still be removed.

More generally, this PR now doesn't really work because other branches have been merged into it that are nothing to do with it (the PR description no longer matches, for example, I assume I'm only reviewing the fixup but that's not clear). Other branches should have been merged into something like  `shropshire-reviewed`, not this branding branch that is the subject of its own PR (and then this PR should also be merging into shropshire-reviewed, and then the whole thing can be tidied and merged to master when ready).

File: templates/web/shropshire/index-steps.html (hunk 1 above)
Comment (MorayMySoc) [reply templates/web/shropshire/index-steps.html:7]: This fix is now in master as was merged from another branch

**Comment by dracos** [comment]
As above, this was merged elsewheres.

**Pull request closed without merging**